packages/backend/src/util/isDevOrStagingUrl.ts

```diff
@@ -0,0 +1,34 @@
+// TODO: use the same function from @clerk/shared once treeshakable
```

Comment:
@nikosdouvlis Did we resolve the tree-shaking issues with `@clerk/shared` yet?

Reply:
#946 should handle the issue with the treeshakable packages
cc @dimkl

Reply:
The mentioned PR is related to the `nextjs` package and not the entire project. This change is inside the `backend` package. Why is that related to the treeshaking of the other PR ?